styles/Home.module.css

```diff
@@ -376,6 +386,137 @@ p.heroTagline:after {
   }
 }
 
+/* Timeline styles */
+.timeline {
```

Comment:
these styles could also be moved to the section folder so to have a cleaner home file

Reply:
Addressed